src/test/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/integTests/framework/api/TestApi.java

```diff
@@ -63,6 +63,16 @@ public JsonElement getRestResponse(@Nonnull final String queryUrl,
     return cluster.getRecordsForAllTablesOnHost(hostTag);
   }
 
+  /**
+   * This API let's a gauntlet test writer swap out the metricsDB for a new one.
+   *
+   * @param clz The class whose AMetric@ should be used to replace it
+   * @throws Exception Throws Exception
+   */
+  public void updateMetrics(Class<?> clz) throws Exception {
+    updateMetrics(clz, false);
+  }
```

Comment:
This diff already exists in mainline. (https://github.com/opendistro-for-elasticsearch/performance-analyzer-rca/blob/59e2f772d666d1d4dd794aacc27714e7679f31cb/src/test/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/integTests/framework/api/TestApi.java#L66)

You might need to merge from master.

Reply:
The change I've made is to create another `updateMetrics` method with only one `clz` parameter and internally calling `updateMetrics(clz, false)` method which you mentioned in master branch.

Reply:
Aah I see.